plugins/woocommerce/includes/class-wc-mobile-messaging-handler.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Prepares formatted mobile deep link navigation link for order mails
+ *
+ * @package WooCommerce\Emails
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Mobile_Messaging_Handler
+ */
+class WC_Mobile_Messaging_Handler {
+
+	const OPEN_ORDER_INTERVAL_DAYS = 30;
```

Comment:
So far, we only use this from within this class. We could consider marking it as private (unless you anticipate referencing it from elsewhere in the near future, of course):

```suggestion
	private const OPEN_ORDER_INTERVAL_DAYS = 30;
```